docs/style-guide.md

```diff
@@ -221,3 +231,47 @@ For an example of when *not* to use lexicographic order, consider an array holdi
 Spec changes that add properties should do so in accordance with this recommendation, rather than automatically placing them at the end—relative enumeration order of properties should remain stable over time, but there is no such expectation regarding adjacency.
 #### Examples
 `Object.keys(new Intl.Segmenter().resolvedOptions())` returns `[ 'locale', 'granularity' ]`. If support for the [Unicode BCP 47 U Extension "dx" key](https://unicode.org/reports/tr35/#UnicodeDictionaryBreakExclusionIdentifier) were added and exposed as `dictionaryBreakExcludedScripts`, that property would belong before `granularity`.
+
+## Naming Conventions
+
+This section provides guidelines for naming conventions in ECMA-402, focusing on clarity, consistency, and adherence to established practices.
+
+### Avoiding Abbreviations in Property Names and Values
+
+:star2: *Property names and values in ECMA-402 should avoid abbreviations to ensure clarity and readability. Use descriptive names that clearly convey their purpose.* :star2:
+
+#### Guidelines
+
+- Use full, descriptive names for properties and values, even if they are longer.
+- Avoid cryptic abbreviations or shortened forms that may confuse readers or developers unfamiliar with the context.
+- Prioritize clarity over brevity, especially for properties that are frequently used or have significant impact.
+
+#### Rationale
+
+- Descriptive names improve code readability and maintainability.
+- They reduce the cognitive load for developers, especially those new to the specification.
+- Avoiding abbreviations aligns with the principle of self-documenting code, making the API easier to understand and use.
+
+#### Exceptions
+
+- Abbreviations may be acceptable if they are widely recognized and standardized (e.g., `UTC`, `ISO`).
+- When abbreviations are used, they should be documented thoroughly to avoid confusion.
+
+### Compound Words in Property Names and Values
+
+:star2: *Combine compound words in property names and values using camelCase.* :star2:
+
+#### Guidelines
+
+- Use camelCase for property names and values that are compound words.
+- Avoid separating words with underscores, hyphens, or other delimiters.
+- For values only, nonzero numbers should be expressed as digits (as in `"h23"`).
+
+#### Rationale
+
+- camelCase is a widely accepted convention in JavaScript and aligns with existing ECMA-402 practices.
```

Comment:
Please dig up the thread in NumberFormat V3 where this was discussed, but we had a number of other reasons. One was that sometimes these strings end up as identifiers, and identifiers in JavaScript are camel case.

Reply:
Is this it? https://github.com/tc39/proposal-intl-numberformat-v3/issues/9

Reply:
Yeah, I thought that was the original issue but I guess it is https://github.com/tc39/proposal-intl-displaynames/issues/32

Actually, the style guide already says to use camel case here. There is a big section explaining the rationale. I think just refer to that previous section.